tests/wpt_tests.cpp

```diff
@@ -146,6 +147,20 @@ bool setters_tests_encoding() {
       if (!element["comment"].get(comment)) {
         std::cout << "    comment: " << comment << std::endl;
       }
+      std::string_view encoding{};
+      ada::encoding_type type = ada::encoding_type::UTF8;
+      if (!element["encoding"].get(encoding)) {
+        std::cout << "    encoding: " << encoding << std::endl;
+        if(encoding == "UTF-8") {
```

Comment:
I think we should have a to_encoding(std::string_view) function for this

Reply:
Agreed.